### ISSUE TITLE
About the range of lambda of isotropic Gaussian blur kernel

### ISSUE THREAD

**Issue by liujianzhao6328057** [JWSoh/MZSR#7]
Thanks for your great job! 
I have read the paper but I cannot find the the the range of lambda when you synthesize the  isotropic Gaussian blur kernel for training.  
I set the sigma to  np.asarray([[lamda, 0],[0, lamda]]) to synthesize the  isotropic Gaussian blur kernel, is that right ?

**Comment by JWSoh** [comment]
It is correct for isotropic Gaussian blur kernels. The range of lambda is from 1.0 to 5.0 for scaling factor (X2). We randomly sampled the lambda when generating the blur kernel for training.

**Comment by BassantTolba1234** [comment]
Please @liujianzhao6328057  I'm facing a problem when i load the pretrained model , specially when it reads the checkpoint
this is the error .. how did you kindly solve it please ??

NotFoundError (see above for traceback): Key MODEL/conv7/kernel/Adam_3 not found in checkpoint
[[Node: save/RestoreV2_69 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_69/tensor_names, save/RestoreV2_69/shape_and_slices)]]

**Comment by BassantTolba1234** [comment]
Please @JWSoh  I'm facing a problem when i load the pretrained model , specially when it reads the checkpoint
this is the error .. how did you kindly solve it please ??

NotFoundError (see above for traceback): Key MODEL/conv7/kernel/Adam_3 not found in checkpoint
[[Node: save/RestoreV2_69 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_69/tensor_names, save/RestoreV2_69/shape_and_slices)]]

**Comment by BassantTolba1234** [comment]
Please can you kindly explain me how to calculate this weight loss ?   

def get_loss_weights(self):
        loss_weights = tf.ones(shape=[self.TASK_ITER]) * (1.0/self.TASK_ITER)
        decay_rate = 1.0 / self.TASK_ITER / (10000 / 3)
        min_value= 0.03 / self.TASK_ITER

        loss_weights_pre = tf.maximum(loss_weights[:-1] - (tf.multiply(tf.to_float(self.global_step), decay_rate)), min_value)

        loss_weight_cur= tf.minimum(loss_weights[-1] + (tf.multiply(tf.to_float(self.global_step),(self.TASK_ITER- 1) * decay_rate)), 1.0 - ((self.TASK_ITER - 1) * min_value))
        loss_weights = tf.concat([[loss_weights_pre], [[loss_weight_cur]]], axis=1)
        return loss_weights
